app/helpers/works_helper.rb

```diff
@@ -96,12 +96,16 @@ def marked_for_later?(work)
     reading && reading.toread?
   end
 
+  def markasread_link(work)
+    link_to ts("Mark as Read"), marktoread_work_path(work)
+  end
+
   def marktoread_link(work)
     link_to ts("Mark for Later"), marktoread_work_path(work)
   end
 
-  def markasread_link(work)
-    link_to ts("Mark as Read"), marktoread_work_path(work)
+  def removemarktoread_link(work)
```

Comment:
Mark as Read, which incorrectly uses the old path (line 99-101)

Why do you think this is using the wrong path ?